README.md

```diff
@@ -1,3 +1,39 @@
 # API-Monitor
 [![CircleCI](https://circleci.com/gh/Ashaba/API-Monitor/tree/develop.svg?style=svg)](https://circleci.com/gh/Ashaba/API-Monitor/tree/develop)
 [![Coverage Status](https://coveralls.io/repos/github/Ashaba/API-Monitor/badge.svg?branch=develop)](https://coveralls.io/github/Ashaba/API-Monitor?branch=develop)
+ 
+ An application the performs health checks of an api
+ 
+ # Dependencies
+ 1. `Python 3.6.2`
+ 1. `PostgreSQL`
+ 1. `JQuery-3.3.1`
+ 1. `Flask 0.12.2`
+ 
+ # Installation
+ 1. Clone the repository  
+ `$ git clone https://github.com/Ashaba/API-Monitor.git`
+ 1. Copy contents of  `env.sample` into `.env`   
+ `$ cp env.sample .env`
+ 1. Configure and setup database in the `.env` file `SQLALCHEMY_DATABASE_URI`  
+ 1. Create a virtual environment  
+ `$ virtualenv -p python3 envname`
+ 1. Activate the virtual environment  
+ `$ source/envname/bin/activate`
+ 1. Install requirements  
+ `$ pip install -r requirements.txt`
+ 1. Add environment variables to the `PYTHONPATH`  
+ `$ export $(cat .env)`
+ 
+ Database Migrations
```

Comment:
This should be a heading. Prepend a **#**.

Reply:
noted, thanks